src/appComponents/sessions/SessionsController.js

```diff
@@ -19,26 +19,27 @@ sessionsModule.config(["$stateProvider", function($stateProvider) {
 sessionsModule.controller("SessionsController",
     ["$mdDialog", "$sessionStorage", "$rootScope", "ApiJsonFactory",
         function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-            var sc = this;
-            if ($sessionStorage.sessions === null ||
-                typeof ($sessionStorage.sessions) === "undefined")
-            {
-                $sessionStorage.sessions = [];
+
+          let sec = this;
+
+          if ($sessionStorage.sessions === null || typeof $sessionStorage.sessions === "undefined") {
+               $sessionStorage.sessions = [];
+
             }
+          sec.showLoaders = false;
+          sec.Sessions = new Array(openevent.totalDays);
+          sec.Days = $sessionStorage.days;
 
-            sc.showLoaders = false;
-            sc.Sessions = new Array(openevent.totalDays);
-            sc.Days = $sessionStorage.days;
+          if ($sessionStorage.sessions.length === 0) {
+              sec.showLoaders = true;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 14 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-webapp/file/2445549891/issues/source?bid=3233857&fileBranchId=3245777#l34)